mllib/src/test/scala/org/apache/spark/ml/classification/GBTClassifierSuite.scala

```diff
@@ -66,10 +66,39 @@ class GBTClassifierSuite extends SparkFunSuite with MLlibTestSparkContext
     ParamsSuite.checkParams(new GBTClassifier)
     val model = new GBTClassificationModel("gbtc",
       Array(new DecisionTreeRegressionModel("dtr", new LeafNode(0.0, 0.0, null), 1)),
-      Array(1.0), 1)
+      Array(1.0), 1, 2)
     ParamsSuite.checkParams(model)
   }
 
+  test("Verify raw scores correspond to labels") {
+    val rawPredictionCol = "MyRawPrediction"
+    val predictionCol = "MyPrediction"
+    val labelCol = "label"
+    val featuresCol = "features"
+    val gbt = new GBTClassifier()
+      .setMaxDepth(2)
+      .setLossType("logistic")
+      .setMaxIter(5)
+      .setStepSize(0.1)
+      .setCheckpointInterval(2)
+      .setSeed(123)
+      .setRawPredictionCol(rawPredictionCol)
+      .setPredictionCol(predictionCol)
+      .setLabelCol(labelCol)
+      .setFeaturesCol(featuresCol)
+    val gbtModel = gbt.fit(trainData.toDF(labelCol, featuresCol))
+    val scoredData = gbtModel.transform(validationData.toDF(labelCol, featuresCol))
+    scoredData.select(rawPredictionCol, predictionCol).collect()
+      .foreach(row => {
```

Comment:
you can use `.foreach { case Row(raw: DenseVector, pred: Double, prob: DenseVector) => ... }` here.